components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -74,227 +159,556 @@ You can see the actions contained in the `editor` role in the [Roles section]({{
-Editor      | editor      | `infra:*`, `compliance:*`, `system:*`, `event:*`, `ingest:*`, `secrets:*`, `telemetry:*`, `applications:*`
-Ingest      | ingest      | `infra:ingest:*`, `compliance:profiles:get`, `compliance:profiles:list`
+Owner              | owner         | \*
+Viewer             | viewer        | secrets:\*:get, secrets:\*:list, infra:\*:get, infra:\*:list, compliance:\*:get, compliance:\*:list, system:\*:get, system:\*:list, event:\*:get, event:\*:list, ingest:\*:get, ingest:\*:list, iam:projects:list, iam:projects:get, applications:\*:list, applications:\*:get
+Editor             | editor        | infra:\*, compliance:\*, system:\*, event:\*, ingest:\*, secrets:\*, telemetry:\*, iam:projects:list, iam:projects:get, iam:projects:assign, applications:\*
+Project Owner      | project-owner | infra:\*, compliance:\*, system:\*, event:\*, ingest:\*, secrets:\*, telemetry:\*, iam:projects:list, iam:projects:get, iam:projects:assign, iam:policies:list, iam:policies:get, iam:policyMembers:\*, iam:teams:list, iam:teams:get, iam:teamUsers:\*, iam:users:get, iam:users:list
+Ingest             | ingest        | infra:ingest:\*, compliance:profiles:get, compliance:profiles:list
 
-### Listing Roles
+**Example Role:**
 
-```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/roles?pretty
+```json
+{
+  "name": "Test Role",
+  "id": "test-role-1",
+  "actions": [
+    "infra:*",
+    "compliance:*",
+    "teams:*",
+    "users:*"
+  ],
+  "projects": [
+    "east-region",
+    "west-region"
+  ]
+}
 ```
 
-The output details each role name, role type, and actions it contains.
+### Listing Roles
+
+The output includes both *Chef-managed* and *Custom* roles.
 
-In the browser: Navigate to the *Roles* page under the **Settings** tab.
+*In the browser:* **Settings**  >> **Roles**
 
 ### Getting a Role
 
-Roles have unique IDs.
-For example, the Editor role's ID is `editor`.
-This may be obtained by first fetching the list of all roles; see [Listing Roles]({{< relref "iam-v2-api-reference.md#listing-roles" >}}).
-Use the role ID as the final path component to fetch a role:
+This shows the details of a single role, selected by its ID.
 
-```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/roles/editor?pretty
+*In the browser:* **Settings**  >> **Roles** >> [select a role]
+
+### Creating a Role
+
+Create a role by composing JSON with all the necessary role properties (see the beginning of the section above).
+
+### Updating a Role
+
+The update operation modifies *all* role properties, not just the ones you specify.
+Properties that you do not specify are reset to empty values.
+Thus, you should supply all of a role's properties, not just the ones you wish to update.
```

Comment:
Again, I'd flip this based on importance: 

>You should supply all of a role's properties, not just the ones you wish to update. Because the update operation modifies *all* role properties, not just the ones you specify, _properties that you do not specify are reset to empty values_.

Reply:
Nice wording! Applied to all sections.